src/BenchmarksServer/Startup.cs

```diff
@@ -560,13 +619,40 @@ private static string GetDotNetExecutable(string dotnetHome)
 
                     if (job.State == ServerState.Starting && e.Data.Contains("Application started"))
                     {
-                        job.State = ServerState.Running;
-                        job.Url = ComputeServerUrl(hostname, job.Scheme, job.Scenario);
                         Log.WriteLine($"Running job '{job.Id}' with scenario '{job.Scenario}'");
+                        job.Url = ComputeServerUrl(hostname, job.Scheme, job.Scenario);
+                        Log.WriteLine("Measuring startup time");
+                        
+                        using(var response = _httpClient.GetAsync(job.Url).GetAwaiter().GetResult())
```

Comment:
Is it correct to wait until `OutputDataReceived` before executing the request?  Is it possible the server can respond to requests before it writes to stdout?

Reply:
Actually, I think the startup and latency code should be moved out of `StartProcess()` completely since they are not related.  I would move the code into `ProcessJobs()` after the call to `StartProcess()`.